### ISSUE TITLE
The mnist training example does not work

### ISSUE THREAD

**Issue by msxingpeng** [Jingkang50/OpenOOD#143]
Thanks for providing this repo!

I downloaded this repo and tried to run the following script, as indicated by the readme file:

`sh scripts/basics/mnist/train_mnist.sh`

However, I got both 11.35% accuracy on testing dataset and training loss is not decreasing!

**Comment by msxingpeng** [comment]
Oh, never mind! I noticed that I should modify the default learning rate.
